### ISSUE TITLE
Incorrect data types for object properties

### ISSUE THREAD

**Issue by jean553** [sourcey/spectacle#4]
Hello,

When in JSON I have :  
"properties": {  
 "platform": {  
   "type": "string",  
   "description": "a description"  
},                          

In the HTML, I get "object" instead of "string" for the attribute "platform". The description and format fields are ignored.

**Comment by LouwHopley** [comment]
Hi, I'm getting the same. Any idea when this will be resolved? :)

**Comment by akamuraasai** [comment]
'm not very deep, but during my tests, I got the same errors. Even if you look at the example available at http://cheesestore.github.io/ see that this error also occurs.
As far as I can trace, it occurs whenever you include a Schema in response to an Operation.

The soulução that I found was to make the following change to the file schemaDatatype.js on line 15:

``` javascript
if (!value.type) {
    if (value.discriminator == false) {
        var str = "";
        for (var key in value){
            if (key != 'discriminator') str += value[key];
        }
        return str;
    }
    return 'object';
}
```

I know it's not the best solution, but that's what I was able to solve my problem.
